sunpy/coordinates/transformations.py

```diff
@@ -789,7 +789,8 @@ def gse_to_gse(from_coo, to_frame):
     if np.all(from_coo.obstime == to_frame.obstime):
         return to_frame.realize_frame(from_coo.data)
     else:
-        return from_coo.transform_to(HeliocentricEarthEcliptic).transform_to(to_frame)
+        return from_coo.transform_to(HeliocentricEarthEcliptic(obstime=from_coo.obstime)).\
```

Comment:
oh come on :stuck_out_tongue_winking_eye:

Reply:
Heh, fine, I'll fix it when I do this for real